relays/ethereum/src/main.rs

```diff
@@ -352,3 +362,17 @@ fn metrics_params(matches: &clap::ArgMatches) -> Result<Option<metrics::MetricsP
 
 	Ok(Some(metrics_params))
 }
+
+fn instance_params(matches: &clap::ArgMatches) -> Result<Box<dyn BridgeInstance>, String> {
+	let instance: Box<dyn BridgeInstance> = if let Some(instance) = matches.value_of("sub-pallet-instance") {
+		match instance.to_lowercase().as_str() {
```

Comment:
Is `to_lowercase` still required if parameter is marked as `case_insensitive` in the yaml file?

Reply:
I'm not sure actually, I'll have to look into it

Reply:
So it looks like `case_insensitive` is just a gatekeeper for what comes through the CLI, but it doesn't actually do anything to the input - like make it lowercase